infrastructure/serve/cosmos.tf

```diff
@@ -0,0 +1,73 @@
+#  Copyright (c) University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+
+resource "azurerm_cosmosdb_account" "serve" {
+  name                = "cosmos-db-${var.naming_suffix}"
+  location            = var.core_rg_location
+  resource_group_name = var.core_rg_name
+  offer_type          = "Standard"
+  kind                = "GlobalDocumentDB"
+  # public_network_access_enabled = var.local_mode
+  tags = var.tags
+
```

Comment:
```suggestion
enable_free_tier = true
```

Reply:
not sure this will work if only one is allowed per sub and there's multiple deployments into our shared dev sub

Reply:
maybe not worth the effort if the standard serverless is basically free(?) 
guess we wouldn't want free in prod because we'd like the same skus in dev to make sure everything works?

Reply:
Well, up to 1000 RUs is free in the free tier, so even with some apps onboarded we should be able to stay within that (for example if we have autoscaled databases set to the 100 RU pre-provisioned minimum and most of them aren't using anything over that we can have 9-10 apps onboarded and still stay mostly free), whereas in the server less model we would be paying for that initial 1000 RU usage

Reply:
As mentioned, happy to leave this out for now and come back to it later